monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -470,15 +484,16 @@ def plan_and_process(
         if not no_pp:
             self.preprocess(c, n_proc, overwrite_plans_name, verbose)
 
-    def train_single_model(self, config: Any, fold: int, gpu_id: int = 0, **kwargs: Any) -> None:
+    def train_single_model(self, config: Any, fold: int, gpu_id: tuple | list | int = 0, **kwargs: Any) -> None:
```

Comment:
@dongyang0122 Is it reasonable to name `gpu_id` as `gpu_ids` as well to be consistent with `train` and `train_parallel`?

Please feel free to ignore if it is not preferred.